### PR TITLE
bump: 2.25.0 -> 2.26.0

### PR DESCRIPTION
**Pull request by semirp**
- bump: Android 2.28.

### CODE REVIEW

File: packages/sdk/android/src/main/java/com/primerioreactnative/PrimerRNHeadlessUniversalCheckout.kt [hunk 1]
```diff
@@ -46,6 +46,7 @@ class PrimerRNHeadlessUniversalCheckout(
         if (settingsStr.isNullOrBlank()) PrimerSettingsRN() else json.decodeFromString(
           settingsStr
         )
+      println(settings)
```
Comment (FlaviuExtPrimer) [inline packages/sdk/android/src/main/java/com/primerioreactnative/PrimerRNHeadlessUniversalCheckout.kt:49]: Is this debugging leftover, do do we keep it?

Comment (semirp) [reply packages/sdk/android/src/main/java/com/primerioreactnative/PrimerRNHeadlessUniversalCheckout.kt:49]: yeah, just testing due to issues with local build

File: packages/example/android/app/src/debug/AndroidManifest.xml [hunk 2]
```diff
@@ -7,7 +7,7 @@
   <application
     android:usesCleartextTraffic="true"
     tools:ignore="GoogleAppIndexingWarning"
-    tools:targetApi="32">
+    tools:targetApi="28">
```
Comment (gt-prime) [inline packages/example/android/app/src/debug/AndroidManifest.xml:10]: [question]: was this causing any issues?

**Review by gt-prime — APPROVED** [review]

File: packages/example/android/app/src/debug/AndroidManifest.xml (hunk 2 above)
Comment (semirp) [reply packages/example/android/app/src/debug/AndroidManifest.xml:10]: Nope

**Pull request merged**